gnucash/gnome-utils/gnc-embedded-window.c

```diff
@@ -194,20 +193,20 @@ gnc_embedded_window_class_init (GncEmbeddedWindowClass *klass)
  *  @param klass A pointer to the class data structure for this
  *  object. */
 static void
-gnc_embedded_window_init (GncEmbeddedWindow *window,
-                          GncEmbeddedWindowClass *klass)
+gnc_embedded_window_init (GncEmbeddedWindow *window)/*,
+                          GncEmbeddedWindowClass *klass)*/
 {
     ENTER("window %p", window);
 
     gtk_orientable_set_orientation (GTK_ORIENTABLE(window), GTK_ORIENTATION_VERTICAL);
 
     // Set the style context for this widget so it can be easily manipulated with css
-    gnc_widget_set_style_context (GTK_WIDGET(window), "GncEmbededWindow");
```

Comment:
Is this in use ? `grep embeded * -Ri` doesn't lead to a result. I think it's save to correct the spelling.